openssl/src/bn.rs

```diff
@@ -665,55 +945,111 @@ impl BigNum {
         }
     }
 
+    /// [`RFC 2409`] defines the Internet Key Exchange.  The prime is in the order
```

Comment:
The documentation here doesn't really tell me anything about what value I should expect to get from this function.